/usr/bin/id: cannot find name for group ID 5026
Traceback (most recent call last):
  File "/data/ling_shared/garciag/SkeletonV1/Test/quadtest.py", line 31, in <module>
    a.purge(threshDistance = (2*np.pi/3000))
  File "/data/ling_shared/garciag/SkeletonV1/Src/DataStruct/DataStructures.py", line 1410, in purge
    r,l,c = leaf.purge(threshDistance = threshDistance)
  File "/data/ling_shared/garciag/SkeletonV1/Src/DataStruct/DataStructures.py", line 1410, in purge
    r,l,c = leaf.purge(threshDistance = threshDistance)
  File "/data/ling_shared/garciag/SkeletonV1/Src/DataStruct/DataStructures.py", line 1410, in purge
    r,l,c = leaf.purge(threshDistance = threshDistance)
  [Previous line repeated 3 more times]
  File "/data/ling_shared/garciag/SkeletonV1/Src/DataStruct/DataStructures.py", line 1446, in purge
    onespace.append(i)
UnboundLocalError: local variable 'onespace' referenced before assignment
